packages/components/src/Status/Status.scss

```diff
@@ -12,10 +12,28 @@ $tc-status-space-btn-icon-label: 5px !default;
 		text-transform: uppercase;
 	}
 
-	> svg {
-		width: $tc-status-icon-size;
-		height: $tc-status-icon-size;
-		margin-right: $tc-status-space-btn-icon-label;
+	&-icon {
+		> svg {
+			width: $tc-status-icon-size;
+			height: $tc-status-icon-size;
+			margin-right: $tc-status-space-btn-icon-label;
+		}
+
+		&.info {
+			color: $brand-info;
+		}
+
+		&.success {
+			color: $brand-success;
+		}
+
+		&.danger {
+			color: $brand-danger;
+		}
+
+		&.muted {
+			color: $brand-muted;
```

Comment:
You have created brand-muted on $grey.
I think just color $grey should be enough here.
$brand-xx are bootstrap-sass variable we have changed using our own colors in the bootstrap theme.
I will not block it for that but I would like other opinion